class-gf-razorpay.php

```diff
@@ -355,6 +355,9 @@ public function post_callback($callback_action, $callback_result)
             $status         = rgar($callback_action, 'type');
         }
 
+        $ref_id    = url_to_postid(wp_get_referer());
+        $ref_title = $ref_id > 0 ? get_the_title($ref_id): "Gravity Form Page";
```

Comment:
In place of "Gravity Form Page" just use "Home Page" . User don't know what is gravity from

Reply:
Correct me if I am wrong,  home page means website home page which is not in this case. Because we are redirecting to the referrer URL from where the Form is submitted.

Reply:
not sure , but it's not user friendly to say "Gravity Form Page" .